### ISSUE TITLE
Review comment for Capsule Project

### ISSUE THREAD

**Issue by Fawke** [Sakthivel/ReactJS-Training-B2#31]
1. Implement react-redux and create some container components.
2. Write unit test cases for your reducers after you create them.
3. Write unit test case for your React components using enzyme.

**Comment by Fawke** [comment]
After 2nd review
1. Implement react-redux and create some container components. [pending]
2. Write unit test cases for your reducers after you create them. [completed]
3. Write unit test case for your React components using enzyme. [completed]

4. Create a public/private channel/room missing. [Funtionality]
5. Identify few more Redux Actions and test them.
6. Configure eslint with airbnb and resolve all lint errors.
7. Multi-tenacy/Multiple-teams/Onboarding pending. [Functionality]

**Comment by Sakthivel** [comment]
@Fawke : I have update so kindly review once again.

**Comment by Fawke** [comment]
1. When sending a message on a channel, it's not appearing immediately at the end, it's appearing after sending sending another message. [BUG]
2. When I click on `CREATE ROOM`, I'm unable to see the created room on the UI. [BUG]
3. On hitting refresh, it's redirecting me to the username page.
4. No way, to navigate between the channels on the UI.
5. Please configure eslint airBnb and resolve all lint errors.
6. CI/CD missing.

**Comment by Sakthivel** [comment]
@Fawke Thanks for the updated.

But I have submitted my Assignment8 for CI/CD still I want me to do the project also?

5. Please configure eslint airBnb and resolve all lint errors. 
Could you please give me some example for it. I have installed here eslint what else I need to do for it to complete.

Let me try to fix bug ASAP.

**Comment by Fawke** [comment]
Yes, CI/CD needs to be implemented for this project, treat this project as a culmination of all the skills and concepts you learned while doing the assignments.

Please refer to this article for airbnb configuration.
https://medium.com/dailyjs/adding-eslint-to-your-project-7bd4feca35a8

**Comment by Sakthivel** [comment]
@Fawke : CI/CD is done. https://sakthi-react-capsule-project.herokuapp.com/

other issues are inprogress..

**Comment by Fawke** [comment]
> 1. When sending a message on a channel, it's not appearing immediately at the end, it's appearing after sending sending another message. [BUG]
> 2. When I click on `CREATE ROOM`, I'm unable to see the created room on the UI. [BUG]
> 3. On hitting refresh, it's redirecting me to the username page.
> 4. No way, to navigate between the channels on the UI.
> 5. Please configure eslint airBnb and resolve all lint errors.
> 6. CI/CD missing.

6. I still don't see a .circle-ci yml file? And ci should be triggered on each push to your git repo, please configure that in git.
5. You've configured eslint, I see (4 errors), please resolve them
1. You've resolved this.

You still need to work on issues, 2, 3 and 4.
